opengovsg/plumber
# PLU-81: [IF-THEN-9005] Remove logic app

## Problem
Logic app is no longer needed.

## Solution
Delete lor.

<img width="1199" alt="Screenshot 2023-10-16 at 12 41 49 PM" src="https://github.com/opengovsg/plumber/assets/135598754/a7c90456-3161-4329-ab56-0d6d6a11bedb">

## Tests
- Verify in prod DB that there are no more steps or execution steps referencing the logic app.
- Check that landing page has no broken icons.
